core/capabilities/ccip/ccipevm/gas_helpers.go

```diff
@@ -27,6 +27,7 @@ const (
 	ExecutionStateProcessingOverheadGas = 2_100 + // COLD_SLOAD_COST for first reading the state
 		20_000 + // SSTORE_SET_GAS for writing from 0 (untouched) to non-zero (in-progress)
 		100 //# SLOAD_GAS = WARM_STORAGE_READ_COST for rewriting from non-zero (in-progress) to non-zero (success/failure)
+	DestGasOverhead = 110_000 + 110_000 + 130_000 // 110K for commit, 110K for RMN, 130K for Exec
```

Comment:
Isn't there no longer any RMN overhead with offchain blessing? (i.e. there's no votes to bless onchain)

Reply:
I think this is from <= V1.5, but we should create a ticket to investigate exact numbers of each of these for V1.6

Reply:
Can we add a comment here indicating that @0xnogo ?